### DISCUSSION TITLE
Formatting and finding coding history

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#219] [Q&A] [answered]
I'm having trouble with the location of my image and the way it influences other formatting.

1. If I move my image to above my contact information, the formatting of my contact information is changed so that ### Contact Information is no longer formatted as a heading and the bullets are no longer bullets. Notice that I've used html to resize my image - do I need a <p> and </p> to delineate the picture from the heading? Is there a different code I should use to do that?

Here's my code:
# My Earth Data Science [Portfolio](https://bioncphilic.github.io)
### Contact Information
* Email: r_neff@southwesterncc.edu
* [LinkedIN](https://www.linkedin.com/in/randi-neff-b7a27823b/)
<img src="/img/003-WYsm.jpg" alt="Snow on mountains in Wyoming in July" width="50%"/>

**Comment by BioNCphilic** [comment]
OK - so my code did not show up and my apologies for doing each point as a thread. I'm having some computer issues and worried I'll lose my work.

**Reply by BioNCphilic** [reply]
Here's the code from a screenshot 
![image](https://github.com/user-attachments/assets/4d46836f-1d22-477a-8f8d-f2f8cee6ed6f)

**Reply by BioNCphilic** [reply]
If I move the image code to after my contact information, I get the picture between my email and my linked in, but they are formatted correctly.
![image](https://github.com/user-attachments/assets/8db3ed77-e680-44e4-86ce-a7e107e224f4)

**Reply by BioNCphilic** [reply, marked as answer]
It looks like the image is automatically added to the bulleted list, because if I change the code a bit, the image is below my contact info, so apparently an empty space will end the bulleted list format. 
![image](https://github.com/user-attachments/assets/57e782b8-c2c5-4fc5-a4e6-95ae2f13c03a)

so now it looks like I've answered my own question because this works!
![image](https://github.com/user-attachments/assets/c9e6bd16-f744-4710-9836-3fb547ead105)

**Comment by nquarder** [comment]
Hi @BioNCphilic and great job troubleshooting this issue. It looks like you answered your own question (which is great!). I'm going to reopen the discussion so that others may benefit from this work. Thank you again for demonstrating how to create a good technical question post and troubleshoot an issue. -NQ
